src/retrace-server-reposync-faf

```diff
@@ -76,12 +76,12 @@ def generate_repo(db: Database, outputdir: Path, opsys: str, release: str, arch:
     repomd = cr.Repomd()
 
     # Add records into the repomd.xml
-    repomdrecords = (("primary",      pri_xml_path, pri_db),
-                     ("filelists",    fil_xml_path, fil_db),
-                     ("other",        oth_xml_path, oth_db),
-                     ("primary_db",   pri_db_path,  None),
-                     ("filelists_db", fil_db_path,  None),
-                     ("other_db",     oth_db_path,  None))
+    repomdrecords = (("primary",      pri_xml_path, pri_db), #pylint: disable=bad-whitespace
```

Comment:
You can actually simply put `#pylint: disable=bad-whitespace` just before the statement and it will apply to the whole block.